src/style.css

```
@@ -0,0 +1,77 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.box {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 33.3%;
+  min-width: 300px;
+  height: 300px;
+  font-size: 100px;
+  color: #fff;
+}
+
+.container > :nth-child(1) {
```

```suggestion
.container > :nth-child(1) {
```
It's not a bad soulution with `:nth-child` but let's create separate class for each block, e.g `box-1`, `box-2` etc. These classes should be responsible for colour and order.
 And use a mix of classes like `class="box box-1"